modules/nf-core/fastqc/main.nf

```diff
@@ -20,6 +20,10 @@ process FASTQC {
 
     script:
     def args = task.ext.args ?: ''
+    def cpu = task.cpus
```

Comment:
Why all these calculations and not just `task.memory`?

Reply:
see below: https://github.com/nf-core/modules/pull/2965#discussion_r1129859990
And aslo when I ran I got the error: 
`Memory value 15000 MB was outside the allowed range (100 - 10000) at /usr/local/bin/fastqc line 203.`

Which I understand as limiting the memory to 10000

Reply:
@matthdsm I'm sure your busy.. When you have time to look at it, I'll be happy